### PR TITLE
Errand wil create tmp shield token

### PR DESCRIPTION
**Pull request by ramonskie**
errand wil now create a temporary shield token for importing

also made the http port of the store job configurable

### CODE REVIEW

File: jobs/import/templates/bin/run [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+$bin/shield api -k https://<%= p("domain") %> <%= p("core") %>
+echo "login to shield"
+$bin/shield login -u <%= p("username") %> -p <%= p("password") %>
+#TODO: create and remove token or keep token.
+echo "creating token"
+token=$($bin/shield create-auth-token tmp)
+echo "$token"
+
+echo "importing now......"
+sed -e "
```
Comment (jhunt) [inline jobs/import/templates/bin/run]: Is this now required?  I.e. you can no longer use a pre-generated token?

Comment (ramonskie) [reply jobs/import/templates/bin/run]: well a pre-generated token isn't necessary anymore with this.
but if you want to give ppl an option i'm okay with that

File: jobs/store/templates/bin/nginx [hunk 2]
```diff
@@ -4,13 +4,15 @@ set -eu
...
+DAT_DIR=/var/vcap/store/webdav
 PIDFILE=$RUN_DIR/nginx.pid
 mkdir -p $LOG_DIR
 exec >>$LOG_DIR/nginx.log 2>&1
 
 case $1 in
   start)
-    mkdir -p $RUN_DIR
+    mkdir -p $RUN_DIR $DAT_DIR
+    chown vcap:vcap $DAT_DIR
```
Comment (jhunt) [inline jobs/store/templates/bin/nginx]: Is this change related?

File: jobs/store/templates/bin/nginx [hunk 3]
```diff
@@ -4,13 +4,15 @@ set -eu
 RUN_DIR=/var/vcap/sys/run/store
 LOG_DIR=/var/vcap/sys/log/store
 JOB_DIR=/var/vcap/jobs/store
+DAT_DIR=/var/vcap/store/webdav
```
Comment (jhunt) [inline jobs/store/templates/bin/nginx]: Is this change related?

Comment (ramonskie) [reply jobs/store/templates/bin/nginx]: is a fix that is already in master
https://github.com/starkandwayne/shield-boshrelease/commit/82894b42c65c2d7ea3cb25d54d9b56e7c5c7a6fb

the store job was not working at all without that fix

**Comment by jhunt** [comment]
I'm going to split this PR into a couple of different commits, since it's really multiple changes in one.
